plugin-infra/go-plugin-access/src/main/java/com/thoughtworks/go/plugin/access/notification/v3/JsonMessageHandler3_0.java

```diff
@@ -24,8 +24,9 @@
 import com.thoughtworks.go.plugin.api.response.Result;
 import org.apache.commons.lang3.NotImplementedException;
 
-import java.text.SimpleDateFormat;
-import java.util.*;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
```

Comment:
Optimizing import in `v3/JsonMessageHandler3_0.java` as nothing has changed in this file. is it ok to undo these import changes?